packages/sequence/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import type { WalletInit } from '@web3-onboard/common'
+
+interface SequenceOptions {
+  appName?: string,
+  network?: number | string
+}
+
+function sequence(options?: SequenceOptions): WalletInit {
+  const { appName = 'app', network = 137 } = options || {}
+
+  return () => {
+    // @ts-ignore
+    return window?.ethereum && window.ethereum?.isSequence ?
+      []
+    :
+     {
+      label: 'Sequence',
+      getIcon: async () => (await import('./icon.js')).default,
+      getInterface: async ({ chains }) => {
```

Comment:
```suggestion
      getInterface: async () => {
```
We can remove the unused reference.

Reply:
Thanks!